stackrox/stackrox
# Remove ROX_QUAY_ROBOT_ACCOUNTS feature flag in ui code

## Description

See https://github.com/stackrox/stackrox/tree/master/ui/apps/platform#delete-a-feature-flag-from-frontend-code

Steps 1 and 2. Skip step 3 because no conditional code in cypress tests.

**Nakul**, can you do step 4 when you remove occurrences from backend?

Delete `ci_export ROX_WHATEVER "${ROX_WHATEVER:-true}"` from `export_test_environment` function in tests/e2e/lib.sh

## Checklist

## Testing Performed

1. `yarn cypress-open` in ui/apps/platform
    * integrations/imageIntegrations.test.js

### manual testing

1. Visit /main/integrations/imageIntegrations/quay and click **Public Quayio** link to see **OAuth token** and **Robot** boxes
    ![Registry_view](https://user-images.githubusercontent.com/11862657/226402088-f7bfc11c-3068-48d2-9e71-4e23101a0c20.png)

2. Click **Edit** button to see information
    ![Registry_edit](https://user-images.githubusercontent.com/11862657/226402111-041c37c6-5ee3-480d-a84d-5241b3ac0498.png)

3. Click **Scanner** to see information and absence of **Robot** boxes
    ![Scanner](https://user-images.githubusercontent.com/11862657/226402145-a495b12f-283b-4c9d-b413-63d85c62c40a.png)

4. Click **Registry + Scanner** to see information
    ![Registry_Scanner](https://user-images.githubusercontent.com/11862657/226402182-5761bcee-32d9-4699-841f-a79623df4e09.png)
